frontend/src/js/services/services.js

```diff
@@ -104,6 +104,53 @@
             angular.element("#sim-loader").fadeOut();
             angular.element(".web-container").removeClass('low-screen');
         };
+
+        this.check = function(password){
+
+           //Regular Expressions.  
+            var regex = new Array();
+            regex.push("[A-Z]"); //Uppercase Alphabet.  
+            regex.push("[a-z]"); //Lowercase Alphabet.  
+            regex.push("[0-9]"); //Digit.  
+            regex.push("[$$!%*#?&]"); //Special Character.  
+            var passed = 0;
+            //Validate for each Regular Expression.  
+            for (var i = 0; i < regex.length; i++) {
```

Comment:
Please replace `regex.length` with `4` (Integer) since length of array is fixed and equal to 4.

Reply:
@spyshiv I think it's good to have dynamic length here as maybe in future if we push another item in regex then we again have to change the length. What do you think?

Reply:
@aka-jain: I agree with your point but what else can be used as password character apart from `[A-Z]`, `[a-z]`, `[0-9]`, `[$$!%*#?&]`.

Reply:
Take in this way..  what if we remove any item from rejex 🤔

Reply:
Ok then `regex.length` is better choice.